.github/workflows/e2e-tests.yml

```diff
@@ -52,14 +52,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes: [v1.20.1,v1.19.2,v1.12.0]
+        kubernetes: [v1.22.1,v1.20.1,v1.19.2,v1.14.0]
```

Comment:
Why the bump in the low range?

Reply:
minikube v1.23.0 's lowest supported version is 1.14.0